config/solid_queue.yml

```diff
@@ -0,0 +1,18 @@
+# default: &default
+#   dispatchers:
+#     - polling_interval: 1
+#       batch_size: 500
+#   workers:
+#     - queues: "*"
+#       threads: 5
+#       processes: 1
+#       polling_interval: 0.1
+#
+# development:
+#  <<: *default
+#
+# test:
+#  <<: *default
+#
+# production:
+#  <<: *default
```

Comment:
Aha - here's that reference to threads. You probably found this, but from what I could tell reading about it yesterday, if you pass no config, it doesn't default to threading.

Reply:
I'm also a lot less worried about threads in the job queue.  I find that the natural way to write jobs is to assuming that you get everything you know from the arguments you're given.  It certainly possible to get bitten by singleton variables like `User.current`, but it wouldn't be meaningful in a job unless you set it in the job which is kind of a silly thing to do.